src/clients/docs_generate.zig

```diff
@@ -0,0 +1,107 @@
+const std = @import("std");
+
+const Docs = @import("./docs_types.zig").Docs;
+const go = @import("./go/docs.zig").GoDocs;
+const node = @import("./node/docs.zig").NodeDocs;
+
+const languages = [_]Docs{go, node};
+
+// pub fn run_in_docker(image: []const u8, cmds: [][]const u8) !void {
+//     var cp = std.child_process.ChildProcess.init("docker", &[_][]const u8{
+//         "run",
+//         language,
+//         },
+//                                                  );
+// }
+
+// pub fn validate_docs(language: Docs) {
+//     try run_in_docker(
+//         language.test_linux_docker_image,
+//         language.install,
+//     );
+//     try run_in_docker(
+//         language.test_linux_docker_image,
+//         language.developer_setup_bash_commands,
+//     );
+// }
+
+pub fn printHeader(writer: std.fs.File.Writer, n: i8, content: []const u8) !void {
+    var x = n;
+    while (x > 0) {
+        try writer.print("#", .{});
+        x -= 1;
+    }
+
+    try writer.print(" {s}\n\n", .{content});
+}
+
+pub fn printParagraph(writer: std.fs.File.Writer, content: []const u8) !void {
+    try writer.print("{s}\n\n", .{content});
```

Comment:
maybe trim content's own `\n` for niceness?

Reply:
Eh no need to add more logic when the reviewer can just look at the diff and leave a comment?